mern-ecommerce/controllers/user.js

```diff
@@ -40,6 +41,32 @@ exports.update = (req, res) => {
   );
 };
 
+exports.create = (req, res) => {
+  const category = new Category(req.body);
+  category.save((err, data) => {
+    if (err) {
+      return res.status(400).json({
+        error: errorHandler(err),
```

Comment:
The `errorHandler` function is not being used correctly. It should be used to handle the error message, not just pass the error object. It should be `error: errorHandler.getErrorMessage(err)`.